superset/db_engine_specs/docs_lib.py

```diff
@@ -0,0 +1,689 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Is this pretty much a duplicate of the script that's building today's markdown file? I'm not sure if we should modify that one, or just add this new one, and remove the old one after this PR merges. 

Also, should this perhaps live in the lower level /scripts folder or somewhere in the /docs directory, perhaps?